Submitted By: Adrian Ireland <aireland AT eircom DOT net>
Date: 2003-10-08
Initial Package Version: 1.4
Origin: http://blfs-bugs.linuxfromscratch.org/show_bug.cgi?id=358
Description: Fix compilation with recent glibc versions.
diff -Naur compface-1.4/cmain.c compface-1.4.1/cmain.c
--- compface-1.4/cmain.c	2000-11-23 00:51:03.000000000 +0000
+++ compface-1.4.1/cmain.c	2003-09-25 09:59:44.000000000 +0100
@@ -26,6 +26,10 @@
 extern char *malloc ();
 #endif	/* STDC_HEADERS */
 
+#include <errno.h>
+
 #ifdef HAVE_FCNTL_H
 #include <fcntl.h>
 #endif
diff -Naur compface-1.4/uncmain.c compface-1.4.1/uncmain.c
--- compface-1.4/uncmain.c	2000-11-23 00:51:24.000000000 +0000
+++ compface-1.4.1/uncmain.c	2003-09-25 10:00:08.000000000 +0100
@@ -19,6 +19,10 @@
 #include "config.h"
 #endif
 
+#include <errno.h>
+
 #ifdef HAVE_FCNTL_H
 #include <fcntl.h>
 #endif
